components/table.scss

```diff
@@ -0,0 +1 @@
+@import "../scss/components/table";
```

Comment:
The table probably needs fonts

Reply:
Yup I think you're right, forgot to add it in.